module/apmelasticsearch/client.go

```diff
@@ -63,17 +63,23 @@ type roundTripper struct {
 func (r *roundTripper) RoundTrip(req *http.Request) (*http.Response, error) {
 	ctx := req.Context()
 	tx := apm.TransactionFromContext(ctx)
+	traceContext := tx.TraceContext()
 	if tx == nil || !tx.Sampled() {
+		apmhttp.SetHeaders(req, traceContext, false)
```

Comment:
Should we default to propagating the legacy header?

Reply:
I think we can skip sending the legacy header. It's only for compatibility with old Elastic APM agents, so not relevant for the Elasticsearch use case where the plan is to extract the traceparent header and use the trace ID in logs.